pom.xml

```diff
@@ -5147,14 +5148,25 @@
             <server.jvm>${env.JAVA8_HOME}</server.jvm>
             <server.jvm.args>${testjvm.commonArgs}</server.jvm.args>
          </properties>
+         <dependencies>
+            <dependency>
+               <groupId>com.sun</groupId>
+               <artifactId>tools</artifactId>
+               <scope>system</scope>
+               <systemPath>${env.JAVA8_HOME}/lib/tools.jar</systemPath>
+            </dependency>
+         </dependencies>
          <build>
             <plugins>
                <plugin>
                   <artifactId>maven-surefire-plugin</artifactId>
                   <configuration>
                      <!-- Run the tests with JDK8 -->
                      <jvm>${env.JAVA8_HOME}/bin/java</jvm>
-                     <argLine>${forkJvmArgs}</argLine>
+                     <argLine>${forkJvmArgs} ${test.jvm.args.extra}</argLine>
+                     <systemProperties>
+                        <test.jvm.version.regex>^1\.8\.0.*</test.jvm.version.regex>
```

Comment:
Maybe we should allow the user to override this property on the command line, so we can also use this profile to compile on one Java 11 JDK and run the tests on another JDK?